schema_based_on_diagram.sql

```diff
@@ -0,0 +1,68 @@
+
+CREATE TABLE patients (
+    id serial PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+
+);
+
+-- invoice table
+
+CREATE TABLE invoices (
+    id serial PRIMARY KEY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP
+    visit_date DATE,
+    medical_history_id INT REFERENCES medical_histories(id)
+);
```

Comment:
- [ Required ] As specified in the `ERD` diagram please add the `payed_at` column in the `invoices` table since this column is part of the `invoices` table definition.